firefox_ui_harness/runtests.py

```diff
@@ -68,9 +68,6 @@ def startTestRunner(runner_class, options, tests):
         if options.installer:
             installer = os.path.realpath(options.installer)
 
-            if not mozinstall.is_installer(installer):
```

Comment:
If this method doesn't do what we expect on windows can we guard this block based on the detected platform rather than skip this validation on all platforms?